common/app/views/fragments/omnitureScript.scala.html

```diff
@@ -0,0 +1,28 @@
+@(item: Option[model.MetaData])
+
+@import conf.Switches.InlineOmnitureScript
+@import conf.Static
+@import conf.Configuration
+@import views.support.OmnitureAnalyticsAccount
+
+@item.map{ item =>
+    <script>
+        @* must always be set before the Omniture file is parsed *@
+        window.s_account = '@OmnitureAnalyticsAccount(item)';
+    </script>
+}
+
+@if(InlineOmnitureScript.isSwitchedOn) {
+    <script>
+        try {
+            @Html(Static.js.omnitureJs)
+        } catch (e) {
+            (new Image()).src = '@{Configuration.debug.beaconUrl}/count/omniture-library-error.gif';
+        }
+    </script>
+} else {
+    @* hardly ever changes, so extremely cacheable *@
+    @* unfortunately we cannot put 'defer' on it because we must be able to guarantee this script has run before other
+    scripts are called, hence the ability to switch between this and an inline version for testing *@
+    <script id="omniture" src="@Static("javascripts/vendor/omniture.js")"></script>
```

Comment:
agreed, it would be great if we could get rid of the blocking, but it's a tricky race condition.  It does feel like the current two options are big compromises for no user benefit, and there should be a third way